revolv/templates/administrator/dashboard.html

```diff
@@ -3,143 +3,148 @@
-                                        {% endif %}
-                                        <li><a class="small button info radius" href="{% url "project:edit" pk=project.id %}">Edit Project</a></li>
-                                    </ul>
-                                </form>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="small-3 columns">
-                        <a href="{% url "project:review" project.pk %}">
-                            <img src="{{ project.preview_photo.url }}"></img>
-                        </a>
-                    </div>
-                    <br/>
-                </div>
-                <div id="repaymentModal{{project.id}}" class="reveal-modal tiny" data-reveal>
-                    <h2>How much was repaid?</h2>
-                    <p class="lead">{{ project.org_name }} has repaid {{ project.amount_repaid }} so far.</p>
-                    <form enctype="multipart/form-data" method="POST" action="{% url "project:review" project.pk %}">
-                        {% csrf_token %}
-                        <div class="row collapse prefix-radius">
-                            <div class="small-1 columns">
-                                <span class="prefix">$</span>
-                            </div>
-                            <div class="small-11 columns">
-                                <input type="text" placeholder="Value" type="number" name="_repayment_amount" pattern="[0-9]+([\.][0-9]+)?" step="0.01" required>
-                            </div>
+      </div>
+
+      <div class="row">    
+          
+        <div class="large-9 push-3 columns">
+          <div class="row homepage-divider">active projects</div><br>
+          <div class='project-container active row' id="active-project-container">
+            {% for project in active_projects %}
+                {% if project == active_projects.first %}
+                      <div class='large-12 pull-0 columns'>
+                        <img src='{{project.cover_photo.url}}'/>
+                        <div class="overlay"></div>
+                        <div class="status-bar">
+                          <a href="#"><p class="status-bar-menu-text">{{project.title}}</p></a>
                         </div>
-                        <input id="repayment_project" class="small button radius" name="_repayment" type="submit" value="Submit"></input>
-                    </form>
-                    <a class="close-reveal-modal">&#215;</a>
-                </div>
-                {% empty %}
-                <p> You have not created any projects. </p>
+                      </div>
+                {% endif %}
             {% endfor %}
-        </div>
-    </div>
-</div>
-
+         </div>
+         <br></br>
+         <div class="row homepage-divider">other projects</div><br>
+         <div id="other-project-container row">
+          {% for project in proposed_projects %}
```

Comment:
In order to make this template cleaner, you should just iterate through all of the projects at once, since there's no different in html for proposed, drafted or completed projects here